ElectroWeakAnalysis/Utilities/python/distTCMET_cfi.py

```diff
@@ -42,7 +42,7 @@
     chi2_tight_max = cms.double(3.0),
     nhits_tight_min = cms.double(11),
     ptErr_tight_max = cms.double(0.1),
-    maxTrackAlgo = cms.int32(8)
+    trackAlgos = cms.vstring("undefAlgorithm", "ctf", "rs", "cosmics", "initialStep", "lowPtTripletStep", "pixelPairStep", "detachedTripletStep"),
```

Comment:
I'm not perfectly happy with ending up to two parameters with similar names (`trackAlgos` and `track_algos`) and different semantics, but I think this issue kind-of existed already before.